.github/workflows/apptainer-attach-on-release.yml

```
@@ -0,0 +1,38 @@
+      - name: Get release version
+        id: get_version
+        run: echo "VERSION=${GITHUB_REF#refs/tags/}" >> $GITHUB_OUTPUT
+
+      - name: Build Apptainer image from Docker URL
+        run: |
+          REPO_LOWERCASE=$(echo "${{ github.repository }}" | tr '[:upper:]' '[:lower:]')
+          VERSION=${{ steps.get_version.outputs.VERSION }}
+          sudo apptainer build app-${VERSION}.sif docker://ghcr.io/${REPO_LOWERCASE}:${VERSION}
+
+      - name: Upload image to release
+        uses: softprops/action-gh-release@v1
```

actionlint (1.7.7)

the runner of "softprops/action-gh-release@v1" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)